war/src/main/webapp/help/parameter/file.html

```diff
@@ -22,6 +22,15 @@
     <code>my.zip</code>
     .) The name will not include the directory name portion.
   </p>
+  <p>
+    Please note that some shells (such as the default /bin/sh) may not be able
```

Comment:
Does this apply to all possible `/bin/sh`? Isn't the underlying implementation OS/distro-dependent?

Reply:
Yes, it depends on OS/distro. So it is just a note if some one also get confused by the unrecognized environment variable `abc.zip`

On Mac OSX, /bin/sh seems link to bash, so `printenv abc.zip` returns the correct result.
But on Ubuntu 22.04, /bin/sh is a symbolic link to dash by default, which do not recognize the "abc.zip" variable.